src/Business Foundation/App/NoSeries/src/Legacy/NoSeriesManagement.Codeunit.al

```diff
@@ -931,11 +971,19 @@ codeunit 396 NoSeriesManagement
     begin
     end;
 
-    [IntegrationEvent(false, false)]
-    internal procedure OnBeforeUpdateNoSeriesLine(var NoSeriesLine: Record "No. Series Line"; NewNo: Code[20]; NewFieldName: Text[100]; var IsHandled: Boolean)
+#if not CLEAN24
+    [Obsolete('This is a temporary method for compatibility only. Please use the "No. Series" codeunit instead', '24.0')]
+    internal procedure RaiseObsoleteOnBeforeUpdateNoSeriesLine(var NoSeriesLine: Record "No. Series Line"; NewNo: Code[20]; NewFieldName: Text[100]; var IsHandled: Boolean)
     begin
+        OnBeforeUpdateNoSeriesLine(NoSeriesLine, NewNo, NewFieldName, IsHandled);
     end;
 
+    [Obsolete('This event is obsolete. Please use the extensibility options provided by the No. Series module.', '24.0')]
```

Comment:
> "Please use the extensibility options provided by the No. Series module."

If the module provides a "twin event", then please name it directly.
Even if such an event does not exist, please point the developer at least to the relevant object(s).

Reply:
There is no direct replacement. As I do not know why or how this event was used I also cannot recommend an alternative. Our data shows that only 1 app is using this event.